src/routes/RequestsRoute.test.js

```diff
@@ -855,6 +878,18 @@ describe('RequestsRoute', () => {
     };
     const requestWithoutData = {};
 
+    describe('select', () => {
+      it('should return select', () => {
+        expect(listFormatter.select(requestWithData)).toBeTruthy();
+      });
+    });
+
+    describe('singlePrint', () => {
+      it('should render "SinglePrintButtonForPickSlip" with correct props', () => {
+        expect(listFormatter.singlePrint({})).toBeTruthy();
+      });
+    });
```

Comment:
Will be mush better to mock components "SinglePrintButtonForPickSlip" and "CheckboxColumn" and then check that those components were rendered.

Reply:
I may need some help with that.